macros/string_mapping.sql

```diff
@@ -0,0 +1,69 @@
+{% macro string_mapping(table_name) -%}
+    {{ return(adapter.dispatch('string_mapping', 'dynamics_365_crm')(table_name)) }}
```

Comment:
Sometimes we use return, sometimes we don't. Do you know why we should/shouldn't? Now is a good time to align on this standard.